examples/apps/echo/client/echo_client.py

```diff
@@ -207,7 +212,14 @@ def Main(args=None):
 		data_encryption_algorithm="AES-GCM-256"
 	)
 	# Add worker input data
-	wo_params.add_in_data(message)
+	if input_data_hash:
+		# Compute data hash for data params inData
+		data_hash = utility.compute_data_hash(message)
+		# Convert data_hash to hex
+		data_hash = ''.join(format(i, '02x') for i in data_hash)
```

Comment:
Don't we have a utility to convert to hex ? If not, can we add it to utility and use it.

Reply:
It makes sense to write a utility function to convert Byte Array to hex string. I can see the same logic duplicated in this file.

Reply:
Ok